kong/clustering.lua

```diff
@@ -343,27 +347,63 @@ local function should_send_config_update(node_version, node_plugins)
                   "automatically once this DP also upgrades to 2.3 or later"
   end
 
-  local minor_cp = KONG_VERSION:match(MAJOR_MINOR_PATTERN)
-  local minor_node = node_version:match(MAJOR_MINOR_PATTERN)
-  if minor_cp ~= minor_node then
-    return false, "version mismatches, CP version: " .. minor_cp ..
-                  " DP version: " .. minor_node,
+  local major_cp, minor_cp = KONG_VERSION:match(MAJOR_MINOR_PATTERN)
+  local major_node, minor_node = node_version:match(MAJOR_MINOR_PATTERN)
+  minor_cp = minor_cp and tonumber(minor_cp) or INF
+  minor_node = minor_node and tonumber(minor_node) or -INF
```

Comment:
Are those conditions really necessary? We have never released a Kong core without minor version since the introduction of Hybrid mode.